### PR TITLE
Update warning templates

### PR DESCRIPTION
**Pull request by DIRECTcut**
Update warning templates to use config options (instead of CLI or runner options).

### CODE REVIEW

File: src/errors/runtime/templates.js [hunk 1]
```diff
@@ -123,6 +123,6 @@ export default {
     [RUNTIME_ERRORS.browserConnectionError]:               '{originErrorMessage}\n{numOfNotOpenedConnection} of {numOfAllConnections} browser connections have not been established:\n{listOfNotOpenedConnections}\n\nHints:\n{listOfHints}',
     [BrowserConnectionErrorHint.TooHighConcurrencyFactor]: 'The error can be due to a concurrency factor that is too high for the host machine’s performance (the factor value {concurrencyFactor} was specified). ' +
                                                            'Try to decrease the concurrency factor or ensure more system resources are available on the host machine.',
-    [BrowserConnectionErrorHint.UseBrowserInitOption]: 'Use the \'--browser-init-timeout\' option to allow more time for the browser to start. The timeout is set to {browserInitTimeoutMsg}.',
+    [BrowserConnectionErrorHint.UseBrowserInitOption]: 'Use the \'browserInitTimeout\' option to allow more time for the browser to start. The timeout is set to {browserInitTimeoutMsg}.',
```
Comment (miherlosev) [inline src/errors/runtime/templates.js:126]: Use double quotes here:
`'Use the "browserInitTimeout"...`

File: src/errors/test-run/templates.js [hunk 2]
```diff
@@ -373,7 +373,7 @@ export default {
     `,
 
     [TEST_RUN_ERRORS.multipleWindowsModeIsDisabledError]: err => `
-        Multi window mode is disabled. Remove the "--disable-multiple-windows" CLI flag or set the "disableMultipleWindows" option to "false" in the API to use the "${err.methodName}" method.
+        Multi window mode is disabled. Disable the "disableMultipleWindows" option to use the "${err.methodName}" method.
```
Comment (miherlosev) [inline src/errors/test-run/templates.js:376]: `Disable the "disableMultipleWindows"...`

It's tautology. 

What about such variant:
```js
Multi window mode is disabled. To use the "${err.methodName}" method you should remove the "disableMultipleWindows" option.
```

**Review by miherlosev — CHANGES_REQUESTED** [review]

File: src/errors/runtime/templates.js (hunk 1 above)
Comment (DIRECTcut) [reply src/errors/runtime/templates.js:126]:
```suggestion
    [BrowserConnectionErrorHint.UseBrowserInitOption]: 'Use the \"browserInitTimeout\" option to allow more time for the browser to start. The timeout is set to {browserInitTimeoutMsg}.',
```

File: src/errors/test-run/templates.js (hunk 2 above)
Comment (DIRECTcut) [reply src/errors/test-run/templates.js:376]:
```suggestion
        Multi-window mode is disabled.  To use the "${err.methodName}" method, remove the "disableMultipleWindows" option.
```

File: src/errors/runtime/templates.js [hunk 3]
```diff
@@ -123,6 +123,6 @@ export default {
     [RUNTIME_ERRORS.browserConnectionError]:               '{originErrorMessage}\n{numOfNotOpenedConnection} of {numOfAllConnections} browser connections have not been established:\n{listOfNotOpenedConnections}\n\nHints:\n{listOfHints}',
     [BrowserConnectionErrorHint.TooHighConcurrencyFactor]: 'The error can be due to a concurrency factor that is too high for the host machine’s performance (the factor value {concurrencyFactor} was specified). ' +
                                                            'Try to decrease the concurrency factor or ensure more system resources are available on the host machine.',
-    [BrowserConnectionErrorHint.UseBrowserInitOption]: 'Use the \'--browser-init-timeout\' option to allow more time for the browser to start. The timeout is set to {browserInitTimeoutMsg}.',
+    [BrowserConnectionErrorHint.UseBrowserInitOption]: 'Use the \"browserInitTimeout\" option to allow more time for the browser to start. The timeout is set to {browserInitTimeoutMsg}.',
```
Comment (miherlosev) [inline src/errors/runtime/templates.js:126]: Remove slashes:
`'Use the "browserInitTimeout" option to...`

**Review by miherlosev — CHANGES_REQUESTED** [review]

File: src/errors/runtime/templates.js (hunk 3 above)
Comment (DIRECTcut) [reply src/errors/runtime/templates.js:126]:
```suggestion
    [BrowserConnectionErrorHint.UseBrowserInitOption]: 'Use the "browserInitTimeout" option to allow more time for the browser to start. The timeout is set to {browserInitTimeoutMsg}.',
```

**Review by miherlosev — APPROVED** [review]

**Pull request merged**